drivers/ethernet/eth_enc28j60.c

```diff
@@ -510,6 +510,7 @@ static int eth_enc28j60_rx(struct device *dev, uint16_t *vlan_tag)
 	struct eth_enc28j60_runtime *context = dev->driver_data;
 	uint16_t lengthfr;
 	uint8_t counter;
+	uint8_t dummy[100];
```

Comment:
This is not good, we are using 100 bytes from stack here. Can't you just use the size of the buffer that is more suitable, it seems that we are reading max 4 bytes below, so why not have that.

Reply:
Sorry for that, spilled debug code, will change back to 4.